modules/ROOT/pages/install-self-managed.adoc

```diff
@@ -16,7 +16,7 @@ Before installing Anypoint Runtime Fabric in a self-managed Kubernetes environme
 * You have installed and configured your Kubernetes environment as follows:
 +
 - Running an EKS, AKS, or GKE Kubernetes environment. Other Kubernetes environments are not supported.
-- Running Kubernetes version 1.17.x to 1.20.x.
+- Running Kubernetes version 1.19.x, 1.20.10 or later, or 1.21.3 or later.
```

Comment:
i think this might not be correct as it could mean we also support 1.22.. etc which we have not tested yet.

Reply:
@rohit-verma7 I'll fix it to specifically say only 1.21.3

Reply:
No i think we will have to say like 1.21.x but we can confirm from @kmendiratta-crm ?

Reply:
I believe previously @kmendiratta-crm said it had to be 1.21.3